src/components/button/button.css

```diff
@@ -0,0 +1,21 @@
+.button-container {
+    height: 100%;
+    flex: 1;
+    box-sizing: border-box;
+}
+
+.button {
+    width: 100%;
+    height: 100%;
+    font-size: 50px;
+    border: 1px solid #fff;
+}
```

Comment:
why put a WHITE border ??, that's why button looks like there's no border......I think you should give it another color
![Screen Shot 2020-03-18 at 11 19 47 PM](https://user-images.githubusercontent.com/40712459/77012593-89f3d980-696e-11ea-861b-865e2e5d8035.png)